src/store/index.js

```diff
@@ -90,4 +99,33 @@ fb.auth.onAuthStateChanged(user => {
   }
 });
 
+/**
+ * GEOLOCATION
+ * ===========
+ *
+ * Firstly, this will prompt the user for permissions to retrieve geolocation.
+ * We want to retrieve a high accuracy position, and it can be a maximum of a
+ * minute old (cache might return cached pos). We also want to time out after 5
+ * seconds as browser geolocation APIs are quite buggy, and sometimes will just
+ * _die_.
+ *
+ * The fallback address is actually my home address, so if people aren't cool
+ * enough to enable geolocation, they'll be inclined to deliver to me first.
+ *
+ * This should probably be changed to something better for humanity, but it'll
+ * do for now.
+ */
+navigator.geolocation.getCurrentPosition(
+  ({ coords: { latitude, longitude } }) => {
+    store.commit("SET_GEOLOCATION", { latitude, longitude });
+  },
+  err => {
+    console.error(
+      `Could not retrieve geolocation. ${err.code}: ${err.message}.`
+    );
+    store.commit("SET_GEOLOCATION", { latitude: 63.41002, longitude: 10.4153 });
```

Comment:
please set the fallback to Trondheim city center as we are planning to release in Trondheim first

Reply:
Selv om teksten er alt for bra

Reply:
okidoki @Adrianht fikser det

Reply:
<3 ser veldig bra ut ellers :)